client/components/modals/ReactionList/ReactionList.tsx

```diff
@@ -0,0 +1,48 @@
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement } from 'react';
+
+import { openUserCard } from '../../../../app/ui/client/lib/UserCard';
+import { IUser } from '../../../../definition/IUser';
+import { useTranslation } from '../../../contexts/TranslationContext';
+import GenericModal from '../../GenericModal';
+import Reactions from './Reactions';
+
+type ReactionListProps = {
+	rid: string;
+	reactions: ReactElement;
+	tabBar: {
+		openUserInfo: (username: IUser['username']) => void;
+	};
+	onClose: () => void;
+};
+
+const ReactionList = ({ rid, reactions, tabBar, onClose }: ReactionListProps): ReactElement => {
+	const t = useTranslation();
+
+	const onClick = useMutableCallback((e) => {
+		const { username } = e.currentTarget.dataset;
+
+		if (!username) {
+			return;
+		}
+
+		openUserCard({
+			username,
+			rid,
+			target: e.currentTarget,
+			open: (e: React.MouseEvent<HTMLElement>) => {
+				e.preventDefault();
+				onClose();
+				tabBar.openUserInfo(username);
```

Comment:
This tabBar is received as a prop of room. Shouldn't we update this and use "useTabBarOpenUserInfo" in place of this?

Reply:
Already discussed